presidio-analyzer/pyproject.toml

```diff
@@ -34,6 +34,9 @@ stanza = { version = "*", optional = true }
 spacy_stanza = { version = "*", optional = true }
 azure-ai-textanalytics = { version = "*", optional = true }
 azure-core = { version = "*", optional = true }
+torch = "^2.4.1"
```

Comment:
@SharonHart these are added when the transformers container is mounting the folder, any idea how to revert it in such case? is there a "mode" or something we can use?